.github/workflows/docker-hub.yml

```
@@ -19,7 +19,7 @@ jobs:
   publish:
     needs: hadolint
     name: Publish
-    runs-on: ubuntu-latest
+    runs-on: zeitgeist-runner
```

```suggestion
    runs-on: [self-hosted, zeitgeist-runner]
```

> Although the self-hosted label is not required, we strongly recommend specifying it when using self-hosted runners to ensure that your job does not unintentionally specify any current or future GitHub-hosted runners.

https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/using-self-hosted-runners-in-a-workflow#about-self-hosted-runner-labels